packages/webex/src/calling.js

```diff
@@ -68,6 +70,29 @@ class Calling extends EventEmitter {
       });
   }
 
+  async deregister() {
+    if (!this.registered) {
+      this.log.info('Authentication: deregister already done', logContext);
```

Comment:
These logs should reflect which deregister we are doing. We should update the test to say webex.internal.device.unregister succesful

Reply:
Done. Updated the logs appropriately. Please check.